test/testdata/generate.sh

```diff
@@ -132,3 +137,4 @@ scaffold_test_project project-v3-addon --plugins="go/v3,declarative"
 scaffold_test_project project-v3-config --component-config
 scaffold_test_project project-v3-v1beta1
 scaffold_test_project project-v3-with-kustomize-v2 --plugins="kustomize/v2-alpha,base.go.kubebuilder.io/v3"
+scaffold_test_project project-v3-with-deploy-image --plugins="go/v3,deploy-image/v1-alpha"
```

Comment:
It is not working because we have not the implementation for the init on this plugin 
So let's call the `--plugins="go/v3,deploy-image/v1-alpha"` when we create api above and remove it from here.